plugins/docker/dune

```diff
@@ -16,5 +16,5 @@
    ppx_deriving_yojson.runtime
    result)
  (preprocess (per_module
-              ((pps ppx_deriving.std ppx_deriving_yojson) pull build run tag push service)
+              ((pps ppx_deriving.std ppx_deriving_yojson) pull build run pread tag push service)
```

Comment:
(we could probably get rid of this per-module thing now - it was a hack for before 4.08, which we no longer support)

Reply:
Should I change this in this PR or in a separate PR?